spec/unit/ridley/sandbox_uploader_spec.rb

```diff
@@ -49,6 +49,34 @@
 
     before { connection.stub(foss?: false) }
 
+    context " when proxy environment variables are set" do
+      let(:checksums) do
+        {
+          chk_id => {
+            url: "http://jsonplaceholder.typicode.com/posts/1",
```

Comment:
I don't love having to use this site for mocked json, but I just needed valid JSON returned, and the test is about proxy settings, not the json payload. If I completely mock the request, it defeats the purpose of testing the proxy settings. Maybe I've just been staring at this for too long.

That being said, I'd love suggestions on how to handle this better in a unit test.